aws-gitlab-main.yml

```diff
@@ -374,12 +438,54 @@ Resources:
       Engine: redis
       EngineVersion: '5.0.6' #newest
       Port: 6379 
-      CacheNodeType: !Ref CacheNodeTypeParameter
+      CacheNodeType: !Ref GitLabCacheNodeTypeParameter
       CacheSubnetGroupName: !Ref GitLabRedisSubnetGroup
       PreferredAvailabilityZones: 
         - !GetAtt GitLabPrivateSubnetA.AvailabilityZone 
       AZMode: single-az #does multi-az == multi-AZ auto-failover?
       #had to put only one Az because had to choose 1 NumCacheNodes when using Redis  
       VpcSecurityGroupIds: 
         - !GetAtt GitLabRedisSecurityGroup.GroupId
-  
\ No newline at end of file
+ 
+  #BASTION HOSTS
+  GitLabBastionHostA:
+    Type: AWS::EC2::Instance
+    Properties: 
+      AvailabilityZone: !Ref BastionHostAAvailabilityZoneParameter 
+      
+      ImageId: !Ref   GitLabBastionHostAAMIParameter #ubuntu server 18.04 LTS (HVM) AMI 
+      InstanceType: !Ref GitLabBastionHostAInstanceTypeParameter
+      SubnetId: !Ref GitLabPublicSubnetA
+      #NetworkInterfaces:
+      #   - AssociatePublicIpAddress: "true" #enable auto-assign public IP 
+      # BlockDeviceMappings:  #/dev/sdh or xvdh? 
+      #   - DeviceName: 
+      #     Ebs:
+      #       VolumeSize: 8 #asks for an additional 8GB root volume; how?
```

Comment:
For storage, we’ll leave everything as default and only add an 8GB root volume. We won’t store anything on this instance. 

**It ain't asking for extra 8, it says leave the default 8...** remember instructions are meant for eyes on console, so if you were to go through this in console you would see the 8.

To answer your question though it would be through BlockDeviceMappings parameter